apps/openmw/mwstate/quicksavemanager.hpp

```diff
@@ -0,0 +1,42 @@
+#ifndef GAME_STATE_QUICKSAVEMANAGER_H
+#define GAME_STATE_QUICKSAVEMANAGER_H
+
+#include <string>
+
+#include "character.hpp"
+#include "../mwbase/statemanager.hpp"
+
+namespace MWState{
+    class QuickSaveManager{
+        std::string mSaveName;
+        int mMaxSaves;
```

Comment:
Shouldn't those `int` be unsigned?

Reply:
Since this value is configured by the settings manager, and that only has the option to return a signed int, I didn't give the type of this int any thought.  However, it does make sense that mMaxSaves should never be negative, because negatives don't make any sense in the context of the class.

Reply:
Does it really matters?
Nobody really needs 2.2 million+ slots for quicksaving.

Reply:
It doesn't matter in practice, but if I'm calling a function, and I see that it takes an int, I will assume that negative values are allowed, so I might call it with a negative number.  If someone with no understanding of QuickSaveManager's implementation just looks at my function declaration, I don't want them to draw any incorrect conclusions because I allow signed ints.  By switching to unsigned, I can make it that much easier for people in the future to understand the code.